examples/trip.rs

```diff
@@ -118,49 +120,101 @@ async fn book_trip(
     }
 }
 
-/// Builds the saga template to book a trip.  A template describes the actions
+/// Registers all actions for the saga
+fn make_trip_action_registry() -> Arc<ActionRegistry<TripSaga>> {
+    let mut registry = ActionRegistry::new();
+
+    // Create a start action capable of returning saga parameters. Each saga or subsaga
```

Comment:
How does this get enforced?  i.e., what happens if you don't do this?

Reply:
The only enforcement is at runtime. Nodes that require this data will have their lookups fail.

Reply:
This is no longer necessary with the changes from @davepacheco